tests/test_app.py

```diff
@@ -313,3 +314,33 @@ def test_get_kernels_dict(mock_run: Mock) -> None:
     assert get_kernels_dict() == {"python": "/some/path", "R": "/other/path"}
     with pytest.raises(ValueError):
         get_kernels_dict()
+
+
+def test_create_project_folder() -> None:
+    """Fails if copytree is called when an error is raised by a functions inside create.
+
+    Fails if copytree is not called when no error is raised.
+    """
+    with patch(
+        f"{PKG}.create_project_from_template", return_value=None
+    ) as mock_template:
+        with patch(f"{PKG}.make_and_init_git_repo", return_value=None):
+            with patch(f"{PKG}.poetry_install", return_value=None):
+                with patch(
+                    f"{PKG}.install_ipykernel", return_value=None
+                ) as mock_kernel:
+                    with patch(f"{PKG}.copytree", return_value=None) as mock_copy:
```

Comment:
These mocks could be put into fixtures to make the tests cleaner and make it easier to write multiple smaller tests

Reply:
Set up a test class with mocks, should be easier to add more tests for the create function now.